### PR TITLE
Add Nitro Type

### PR DESCRIPTION
**Pull request by theLMGN**

### CODE REVIEW

File: Nitro Type/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
...3%202.1%202.4%204.3%202.4%205%200%20.7-2%2011.6-4.5%2024.3-2.5%2012.6-4.8%2024.6-5.1%2026.5-.3%201.9-1%205.5-1.6%208-.5%202.5-1.2%205.6-1.4%207-.3%201.4-1.1%206-2%2010.3-.8%204.2-1.8%209-2%2010.5-.3%201.5-1.4%207.2-2.5%2012.7-1.1%205.5-2.2%2011.1-2.5%2012.5-.2%201.4-.7%203.5-.9%204.8L26%20231l96.2-.2%2096.3-.3%201.6-8c.9-4.4%202.4-11.6%203.4-16%20.9-4.4%203.9-18.8%206.6-32%202.8-13.2%205.5-26.3%206-29%20.6-2.8%201.3-5.9%201.5-7%20.2-1.1.8-4%201.4-6.5.9-4.1%2010.2-49.1%2011.4-55%20.2-1.4%202.5-12.3%205-24.4%202.5-12%204.6-22.2%204.6-22.7s-55.5-.9-130.1-.9H-.2l2%203.2zm141.1%2060.1c20%2033%2019.2%2032.1%2020.5%2022.2.2-1.7%201.2-6.4%202-10.5.9-4.1%202.3-10.9%203-15%202.7-14.7%205-25.5%205.8-26.6%201-1.4%2042.4-1.3%2042.2.1-.4%202.1-6.8%2035.1-12.9%2066-3.4%2017.6-7.7%2039.5-9.5%2048.7-2.9%2014.9-3.5%2016.7-5.4%2017-1.1.2-9.3.3-18.1.3l-16-.1-8.3-13.4c-4.5-7.4-8.8-14.5-9.5-15.8-3.8-6.9-19.2-32-19.5-31.7-.3.3-3.8%2016.5-4.8%2022-.8%205.2-5.4%2028.8-6.5%2033.7-.6%202.5-1.6%204.8-2.2%205-1.7.6-40.5.3-41.1-.3-.2-.2.7-6%202-12.9%201.4-6.9%202.7-13.6%202.9-15%20.2-1.4%203.8-19.4%207.9-40%204.1-20.6%208.4-42.7%209.6-49%203.4-17.6.9-15.7%2021.6-15.6l17.7.1%2018.6%2030.8z%22%2F%3E%3C%2Fsvg%3E",
```
Comment (Bas950) [inline Nitro Type/dist/metadata.json]: Version should be 1.0
Please upload logo to Imgur

File: Nitro Type/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "author": {
+      "name": "theLMGN",
+      "id": "158311402677731328"
+    },
+    "contributors": [],
```
Comment (Bas950) [inline Nitro Type/dist/metadata.json]: remove contributors as it is optional and you dont use it

**Comment by theLMGN** [comment]
@Bas950 How would you like me to do that? I have the presence enabled on Discord (right now probably)

**Comment by theLMGN** [comment]
image
image

image

**Pull request merged**